server/routes/mentions.js

```diff
@@ -0,0 +1,76 @@
+const router = require("express").Router();
+
+const requiresAuth = require("./middleware/requiresAuth");
+const { Mention, User } = require("../models");
+
+const callScraper = require("../scraper");
+
+// TODO test mentions in action
+
+router.get("/", requiresAuth, async (req, res) => {
+  const user = await User.findByPk(req.user.id);
+  // note that the getter method for n-m can accept options
+  // as if it were a regular query
+  const mentions = await user.getMentions();
+
+  res.json(mentions);
+});
+
+router.get("/:id", requiresAuth, async (req, res) => {
+  const mention = await Mention.findByPk(req.params.id);
+
+  res.json(mention);
+});
+
+router.post("/", requiresAuth, async (req, res) => {
+  // NOTE I think this is better served as just a test route in order to test the scraper
+  // This shouldn't be placed in the final PR as there will be another means to post new data
+  // into the database.
+  const user = await User.findByPk(req.user.id);
+
+  let mentions = callScraper(req.user.company, 100);
```

Comment:
I think, when we want to add new mentions to DB, we have 2 logics:
1. scrapes based on a user's company, which is when we send the weekly email.
2. scrapes based on a user's input in the search bar, which is when the user uses the main page.

Both of them requires only a company name string, so maybe in this router we only want `callScraper(req.companyName, 100)` and let the caller of this router determine what params should be passed.

I noteiced that this is a test router, so if you were already about to change it this way, please ignore this comment.